retrofit-converters/logansquare/README.md

```diff
@@ -0,0 +1,8 @@
+LoganSquare Converter
+=================
+
+A `Converter` which uses [LoganSquare][1] for serialization to and from JSON.
+
+Each type is assigned its corresponding `JsonMapper`, then parsed and serialized.
```

Comment:
The second paragraph of other converters focuses on details you need to know for configuring. Since this library has no runtime configuration, you end up describing implementation details of the converter here.

Maybe change it to "The `JsonMapper` instances generated by the compiler are used for conversion."